### PR TITLE
docs(sql): separating entity definition

### PR DESCRIPTION
**Pull request by Sikora00**
refs nestjs/typeorm#151

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/nestjs/docs.nestjs.com/blob/master/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Other... Please describe:
```

## What is the current behavior?
There is no documentation of how to work with EntitySchema.
Issue Number: nestjs/typeorm#151

## What is the new behavior?
image

## Does this PR introduce a breaking change?
```
[] Yes
[x] No
```

## Other information
Documentation for nestjs/typeorm#427

### CODE REVIEW

File: content/techniques/sql.md [hunk 1]
```diff
@@ -358,6 +358,64 @@ With that option specified, every entity registered through the `forFeature()` m
...
+    relations: {
+        photos: {
+            type: 'one-to-many',
+            target: 'Photo' // The name of the PhotoSchema
+        }
+    }
+});
+```
+> error **Warning** If you provide the `target` option the `name` option has to be the same as the name of the target class.
+> If you not provide the target you can use any name.
```
Comment (johnbiundo) [inline content/techniques/sql.md:399]:
```suggestion
> If you do not provide the `target` you can use any name.
```

File: content/techniques/sql.md [hunk 2]
```diff
@@ -358,6 +358,64 @@ With that option specified, every entity registered through the `forFeature()` m
...
+
+
+Nest allows you to use EntitySchema instance where Entity is expected, for example:
```
Comment (johnbiundo) [inline content/techniques/sql.md:402]:
```suggestion
Nest allows you to use an `EntitySchema` instance wherever an `Entity` is expected, for example:
```

**Comment by kamilmysliwiec** [comment]
This is no longer blocked. Can you make a final review @johnbiundo? :)

File: content/techniques/sql.md [hunk 3]
```diff
@@ -358,6 +358,64 @@ With that option specified, every entity registered through the `forFeature()` m
...
+    },
+    relations: {
+        photos: {
+            type: 'one-to-many',
+            target: 'Photo' // The name of the PhotoSchema
+        }
+    }
+});
+```
+> error **Warning** If you provide the `target` option the `name` option has to be the same as the name of the target class.
```
Comment (johnbiundo) [inline content/techniques/sql.md:398]:
```suggestion
> error **Warning** If you provide the `target` option, the `name` option value has to be the same as the name of the target class.
```

**Comment by kamilmysliwiec** [comment]
Thanks!

**Pull request merged**